### DISCUSSION TITLE
Syncing my photos and videos

### DISCUSSION THREAD

**Discussion by archally298** [MobiusSync/MobiusSync#219] [Q&A]
Hello,

Can someone elaborate on how we can sync photos and videos on iOS?  I found this FAQ, but every time I try to create a folder and choose the "Pick Photo Collection" option, it only appears to create an empty folder in the file manager with no photos.  

> Syncing my photos and videos
> Because iOS manages photos and videos within the Photo Library, they are not accessible to Syncthing to synchronise directly as files. We have been working on custom code to access the iOS Photo Library.
> 
> Starting with v1.27.9, when adding a sync folder, you can use Pick Photo Folder to sync a collection from the iOS Photo Library. Only Send Only Folder Type is supported at this stage. Please be aware that deletions on your iOS device will be also applied (i.e. deleted) on other devices. You may wish to consider the ignoreDelete advanced folder setting: https://docs.syncthing.net/advanced/folder-ignoredelete.html
> 
> You can follow and influence the growing support of Photo Collection sync at: https://github.com/MobiusSync/MobiusSync/

**Comment by fastalan** [comment]
Same here Dec 2024. Appears there is a permissions issue.

**Reply by fastalan** [reply]
The problem was, you can't test this feature until you pay for the application. Very silly that this is required and not explained very well.

**Comment by s-k-e-t-c-h** [comment]
Hi @archally298 ,
I assume, someone would be able to help after getting to know from you, which devices you have connected.

From my side, I can only tell my experience from today:
I used Möbius Sync on an IPhone 11 and connected it with my Desktop PC which runs LINUX Mint 22 and Syncthing.
I chose in the IPhone to create folders for each month.
It works quite well. ✅ Folders were created with pictures inside. ✅ Timestamps were kept.
![grafik](https://github.com/user-attachments/assets/ac842120-0797-4abd-8a10-9b16a48e6008)

**Comment by SimonPickup** [comment]
> it only appears to create an empty folder in the file manager with no photos.

There should be no folder created in the file manager for a photo folder.

You should:
- choose **_+ Add Folder_**
- choose **_Pick Photo Collection_**
- choose **_Proceed_**
- select a photo collection from the list and press **_Select_**
- switch to the Sharing tab and select the device(s) you wish to share with
- press **Save**

The new folder should appear under your list of Syncthing folders in MöbiusSync showing the number of files/folders found, and be offered for sharing on your other device(s).

Where does your experience diverge?

**Reply by iron-saxon** [reply]
@SimonPickup  

See Shared Photo Library support #234 under Discussion Ideas.  Posted here in case it's a user configuration error:

Choosing "All Photos" from "Select a photo collection" syncs all photos and/or videos from all sources (recents, favs, whatsapp, imports, videos, selfies, live, portrait, etc.). As a result, it is copying not only my current Library of photos/videos, but all kinds of photos/videos that are very old (i.e., 2019??) and are even 0kb. However, if I narrow down the sync folder, i.e., "Videos", then it only syncs the video files to my NAS and appears to be only videos from my Library and not shared libraries (confirmed in testing).

Use Case: While travelling and taking pics on my iPhone, I would like to open up MobiusSync throughout the day to backup my Library of photos/videos in case of iPhone loss of whatever sorts. Need an option to Pick Photo 'Library'.

NOTE: Running v1.28.0-188 for iOS

**Comment by iron-saxon** [comment]
After testing, I figured it out (hopefully user configuration issue only).

Add Folder
    Pick Photo Selection
        Select 'Recents' from the list (not All Photos)
        Go Photo Collection (top of app below General and Sharing) will have Include photos, videos, live photos as photos and edited form all checked

This will sync the files in the iPhone 'Library', which is the view we are used too in the Apple Photos App, and not the other collections or shared libraries.

I also found
- If you take a picture/video with the Camera, it syncs almost immediately
- If you delete a picture/video out of the Library, it removes it from the sync location (for me it's my NAS) almost immediately
